src/SFA.DAS.AssessorService.Web/Views/Home/Index.cshtml

```diff
@@ -1,14 +1,17 @@
-﻿@model HomeIndexViewModel
+﻿@using SFA.DAS.AssessorService.Web.Helpers;
+@model HomeIndexViewModel
 @{
     ViewData["Title"] = "Apprenticeship assessment service";
     ViewBag.ZenDeskLabel = "ass-start-page";
     Layout = "_Layout";
+    Model.Banner = new Banner("~/Views/Banners/_NoAssessBannerPartial.cshtml");
```

Comment:
This should be set where the model is created in the controller

Reply:
Reworked as mentioned above